collectors/pod_test.go

```diff
@@ -267,8 +277,9 @@ func TestPodCollector(t *testing.T) {
 						NodeName: "node1",
 					},
 					Status: v1.PodStatus{
-						HostIP: "1.1.1.1",
-						PodIP:  "1.2.3.4",
+						HostIP:    "1.1.1.1",
+						PodIP:     "1.2.3.4",
+						StartTime: &unversionedStartTime,
```

Comment:
@brancz There are two pods for test `kube_pod_info` and `kube_pod_start_time`. I have only added non `nil` `StartTime` to `pod1` and leaving `StartTime` for `pod2` uses the default value for a pointer `nil`. Thus, we have tested both cases. :)